### PR TITLE
ENH: Add from_arrays for MultiIndex

### PR DESCRIPTION
**Pull request by charlesdong1991**
I was trying to create a MultiIndex from an array just now, and found out it hadnt been implemented, so have this PR to create MI from arrays.

```python
>>> arrays = [['a', 'a', 'b', 'b'], ['red', 'blue', 'red', 'blue']]
>>> ks.MultiIndex.from_arrays(arrays)
MultiIndex([('a',  'red'),
                    ('a', 'blue'),
                    ('b',  'red'),
                    ('b', 'blue')],
                   names=['number', 'color'])
```

### CODE REVIEW

**Comment by itholic** [comment]
```
ks..MultiIndex.from_arrays(arrays, names=('number', 'color'))

2144

2145 ^

2146

2147SyntaxError: invalid syntax

2148

2149/home/travis/build/databricks/koalas/databricks/koalas/indexes.py:595: 
```

Maybe here is the problem ?

`ks..` -> `ks.` :smile:

**Comment by charlesdong1991** [comment]
hahaha, yeah, my bad. I didn't check carefully and then went to bed because i was too sleepy yesterday. @itholic

**Comment by itholic** [comment]
haha. yeah that is also happening to me often. 😄

**Comment by ueshin** [comment]
@charlesdong1991 Shall we keep the example in the first commit with `doctest: +SKIP`?
Otherwise, LGTM.

**Comment by charlesdong1991** [comment]
oh, okay 😅

File: databricks/koalas/indexes.py [hunk 1]
```diff
@@ -569,6 +569,40 @@ def from_tuples(tuples, sortorder=None, names=None):
...
+
+        Returns
+        -------
+        index: MultiIndex
+
+        Examples
+        --------
+
+        >>> arrays = [[1, 1, 2, 2], ['red', 'blue', 'red', 'blue']]
+        >>> ks..MultiIndex.from_arrays(arrays, names=('number', 'color')) # doctest: +SKIP
```
Comment (ueshin) [inline databricks/koalas/indexes.py]: oh, sorry, please fix `ks.Multi`. 😄 
also nit: need 2 spaces before `#`.

Comment (charlesdong1991) [reply databricks/koalas/indexes.py]: oh, god, sorry 😅

**Comment by ueshin** [comment]
Thanks! merging.

**Pull request merged**